src/resources/postcss/base/full/_variables.pcss

```
@@ -90,8 +90,8 @@
 	 * ----------------------------------------------------------------------------- */
 	--tec-color-category-primary: transparent;
 	--tec-color-category-secondary: transparent;
-	--tec-color-category-text: var(--tec-color-accent-primary);
-	--tec-color-category-legend-default-1: #334aff;
+	--tec-color-category-text: var(--tec-color-category-text);
```

Is this one redundant? Or is it supposed to be 
```suggestion
	--tec-color-category-text: var(--tec-color-text-primary);
```